src/utils/capitalize.ts

```diff
@@ -0,0 +1,8 @@
+function titleCaseWord(word: string) {
+  if (!word) {
+    return word;
+  }
+  return word[0].toUpperCase() + word.slice(1);
+}
+
+export {titleCaseWord};
```

Comment:
```suggestion
export const titleCaseWord = (word: string) => {
  if (!word) {
    return word;
  }
  return word[0].toUpperCase() + word.slice(1);
}
```

i prefer writing functions this way, mostly just a style difference.